roles/openshift_etcd_migrate/templates/validatemigration.py.j2

```diff
@@ -0,0 +1,77 @@
+from subprocess import PIPE, Popen
+import logging
+import json
+import base64
+
+#logging.basicConfig(level=logging.INFO)
+
+def runCommand(cmd):
+	process = Popen(cmd, stderr=PIPE, stdout=PIPE, shell=True)
+	stdout, stderr = process.communicate()
+	rt = process.returncode
+
+	return stdout, stderr, rt
+
+def keyExists(etcddata):
+	if "count" not in etcddata:
+		return False
+	if etcddata["count"] < 1:
+		return False
+	if "kvs" not in etcddata:
+		return False
+	if len(etcddata["kvs"]) < 1:
+		return False
+	if "value" not in etcddata["kvs"][0]:
+		return False
+	if "key" not in etcddata["kvs"][0]:
+		return False
+
+	return True
+
+# TODO(jchaloup): once done ping kwoodson to test the etcd migration role
+
+if __name__ == "__main__":
+
+	ectl2="/usr/bin/etcdctl --cert-file {{ etcd_peer_cert_file }} --key-file {{ etcd_peer_key_file }} --ca-file {{ etcd_peer_ca_file }} -C https://{{ etcd_peer }}:2379"
+	ectl3="/usr/bin/etcdctl --cert {{ etcd_peer_cert_file }} --key {{ etcd_peer_key_file }} --cacert {{ etcd_peer_ca_file }} --endpoints 'https://{{ etcd_peer }}:2379'"
+
+	so, se, rc = runCommand("%s ls -r | sort" % ectl2)
+	for line in so.split("\n"):
```

Comment:
This will fail on Python 3 as you can not split ``bytes`` this way. If you want to always use ``stdout`` as a ``str`` you will want to use ``.decode('utf-8')`` in your ``runCommand`` method.